rdr_service/tools/tool_libs/genomic_utils.py

```diff
@@ -493,6 +493,86 @@ def run(self):
         return 0
 
 
+class ManualSampleClass(GenomicManifestBase):
+    """
+    Class for inserting arbitrary genomic samples manually. Used for E2E testing.
+    """
+    def __init__(self, args, gcp_env: GCPEnvConfigObject):
+        super(ManualSampleClass, self).__init__(args, gcp_env)
+
+    def run(self):
+        """
+        Main program process
+        :return: Exit code value
+        """
+
+        # Validate Aruguments
+        if self.args.csv is None:
+            _logger.error('Argument --csv must be provided.')
+            return 1
```

Comment:
Could this be done by requiring the argument on the sub-parser?

Reply:
Done.